projects/plugins/jetpack/class-jetpack-pre-connection-jitms.php

```diff
@@ -118,7 +150,7 @@ private function generate_admin_url( $args ) {
 	 * @return array The array of pre-connection JITMs.
 	 */
 	public function add_pre_connection_jitms( $pre_connection_messages ) {
-		$jetpack_messages = $this->get_raw_messages();
+		$jetpack_messages = array_merge( $this->get_raw_messages(), $this->maybe_get_raw_partnership_messages() );
```

Comment:
Would it be an option to add the message directly into `get_raw_messages()`? Maybe that would make it easier for folks looking to quickly see the whole list of pre-connection JITMs?

Reply:
We could definitely add them in the same location. My main thought for the separation is that "get_raw_messages" would have to contain partner conditions logic as well ("Should we add these extra messages?") and separation of concern became a thing.

I don't mind updating it if you lean just 1% towards this living in the same method instead 😄

Reply:
> I don't mind updating it if you lean just 1% towards this living in the same method instead

Nah, that's alright as it is, it's not that big of a difference. Let's 🚢 this :)